src/sentry/static/sentry/app/utils/discover/traceHoverCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import {Location, LocationDescriptor} from 'history';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import withApi from 'app/utils/withApi';
+import {Client} from 'app/api';
+import Hovercard from 'app/components/hovercard';
+import Version from 'app/components/version';
+import space from 'app/styles/space';
+import Clipboard from 'app/components/clipboard';
+import {IconCopy} from 'app/icons';
+import {TableData} from 'app/views/eventsV2/table/types';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import LoadingError from 'app/components/loadingError';
+
+import EventView from './eventView';
+import DiscoverQuery from './discoverQuery';
+
+type ChildrenProps = {to: LocationDescriptor};
+
+type Props = {
+  api: Client;
+
+  orgId: string;
+  traceId: string;
+
+  location: Location;
+
+  children: (props: ChildrenProps) => React.ReactNode;
+
+  // hover card props
+  containerClassName: string;
+};
+
+class TraceHoverCard extends React.Component<Props> {
+  renderHeader() {
+    const {traceId} = this.props;
+
+    return (
+      <HeaderWrapper>
+        <span>{t('Trace')}</span>
+        <TraceWrapper>
+          <StyledTrace version={traceId} truncate anchor={false} />
+          <Clipboard value={traceId}>
+            <ClipboardIconWrapper>
+              <IconCopy size="xs" />
+            </ClipboardIconWrapper>
+          </Clipboard>
+        </TraceWrapper>
+      </HeaderWrapper>
+    );
+  }
+
+  renderBody({
+    tableData,
+    isLoading,
+    error,
+  }: {
+    tableData: TableData | null;
+    isLoading: boolean;
+    error: null | string;
+  }) {
+    if (isLoading) {
+      return (
+        <LoadingWrapper>
+          <LoadingIndicator mini />
+        </LoadingWrapper>
+      );
+    }
+
+    if (error) {
+      return <LoadingError />;
+    }
+
+    if (!tableData) {
+      return null;
+    }
+
+    const numOfTransactions = tableData?.data.length ?? 0;
+
+    return (
+      <div>
+        <div>
+          <h6>{t('Transactions')}</h6>
+          <div className="count-since">{numOfTransactions}</div>
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    const {traceId, location, api, orgId} = this.props;
+    const traceEventView = EventView.fromNewQueryWithLocation(
+      {
+        id: undefined,
+        name: `Transactions with Trace ID ${traceId}`,
+        fields: [
+          'transaction',
+          'project',
+          'trace.span',
+          'transaction.duration',
+          'timestamp',
+        ],
+        orderby: '-timestamp',
+        query: `event.type:transaction trace:${traceId}`,
+        projects: [],
```

Comment:
If the user doesn't have global-views this will currently not work. You will need to check for `global-views` and select only the issue's project if the organization doesn't have global-views.

Reply:
hmm I tested without global-views and it seems to work because i grab the project from the location parameters via `EventView.fromNewQueryWithLocation`. Is there a case where the issue's project would not be in the url/location? If so, I can pass the project down from `eventTagsPill`

Reply:
Pulling the projects from the location would work.